plugins/modules/rabbitmq_exchange.py

```diff
@@ -124,6 +127,10 @@ def main():
     if not HAS_REQUESTS:
         module.fail_json(msg=missing_required_lib("requests"), exception=REQUESTS_IMP_ERR)
 
+    # exchange plugin type to plugin name mapping
+    exchange_plugins = {'x-consistent-hash': 'rabbitmq_consistent_hash_exchange',
```

Comment:
Do we need to add `x-delayed-message` here?

Reply:
Added.